.evergreen/config.yml

```diff
@@ -449,67 +311,27 @@ axes:
         display_name: Sharded Cluster
         variables:
            TOPOLOGY: "sharded_cluster"
-  - id: auth
-    display_name: Authentication
-    values:
-      - id: auth
-        display_name: Auth
-        variables:
-           AUTH: "auth"
-      - id: noauth
-        display_name: NoAuth
-        variables:
-           AUTH: "noauth"
-  - id: ssl
-    display_name: SSL
-    values:
-      - id: ssl
-        display_name: SSL
-        variables:
-           SSL: "ssl"
-      - id: nossl
-        display_name: NoSSL
-        variables:
-           SSL: "nossl"
-  - id: storage-engine
-    display_name: Storage
+  - id: swift-version
+    display_name: "Swift"
     values:
-      - id: mmapv1
-        display_name: MMAPv1
+      - id: "4.2"
+        display_name: "Swift 4.2"
         variables:
-           STORAGE_ENGINE: "mmapv1"
-      - id: wiredtiger
-        display_name: WiredTiger
+           SWIFT_VERSION: "4.2"
+      - id: "5.0"
+        display_name: "Swift 5.0"
         variables:
-           STORAGE_ENGINE: "wiredtiger"
-      - id: inmemory
-        display_name: InMemory
-        variables:
-           STORAGE_ENGINE: "inmemory"
+           SWIFT_VERSION: "5.0"
+
 
 
 buildvariants:
+
 - matrix_name: "tests-all"
-#  matrix_spec: {"os-fully-featured": "*", auth: "*", ssl: "*" } // reenable when SSL support is added to the driver
-  matrix_spec: {"os-fully-featured": "*", auth: "noauth", ssl: "nossl" }
-  display_name: "${os-fully-featured} ${auth} ${ssl}"
+  matrix_spec: {"os-fully-featured": "*", "swift-version": "*"}
+  display_name: "${swift-version} ${os-fully-featured}"
   tasks:
-     - name: "test-latest-replica_set"
-     - name: "test-latest-sharded_cluster"
-     - name: "test-latest-standalone"
-     - name: "test-3.6-replica_set"
-     - name: "test-3.6-sharded_cluster"
-     - name: "test-3.6-standalone"
-     - name: "test-3.4-replica_set"
-     - name: "test-3.4-sharded_cluster"
-     - name: "test-3.4-standalone"
-     - name: "test-3.2-replica_set"
-     - name: "test-3.2-sharded_cluster"
-     - name: "test-3.2-standalone"
-     - name: "test-3.0-replica_set"
-     - name: "test-3.0-sharded_cluster"
-     - name: "test-3.0-standalone"
-     - name: "test-2.6-replica_set"
-     - name: "test-2.6-sharded_cluster"
-     - name: "test-2.6-standalone"
+     - ".latest"
```

Comment:
does this just do matching based on tags then?

Reply:
yep!